cogs/twitcasting/twitcasting.py

```diff
@@ -0,0 +1,190 @@
+                user_embed.add_field(name=user['name'],
+                                     value=f'ID: {user["user_id"]}',
+                                     inline=False)
+        await ctx.send(embed=user_embed)
+
+    @tc.command()
+    async def remove(self, ctx, user_id: str):
+        """
+        Removes subscription to the provided twitcasting user id.
+            Usage: z!tc remove <twitcasting-user-id>
+        """
+        channel_id = f'{ctx.guild.id}:{ctx.channel.id}'
+        self.connection.delete_record(channel_id=channel_id, user_id=user_id)
+        self.twitcasting.remove_webhook(user_id=user_id)
+        await ctx.send(f"Deleted user {user_id} from list of webhooks.")
+
+    @tc.command()
+    async def rename(self, ctx, webhook_name: str):
+        """
+        Migrates subscriptions to a new webhook with the provided name. Deletes the previous webhook.
+            Usage: z!tc rename <new-webhook-name>
+        """
+        channel_id = f'{ctx.guild.id}:{ctx.channel.id}'
+        res = self.connection.select_records_by_id(channel_id=channel_id)
+        if res:
+            webhook_id = res[0]['webhook_id']
+            try:
+                webhook = await self.bot.fetch_webhook(webhook_id)
+                await webhook.delete()
+            except NotFound:
+                self.logger.warning("Could not find and delete previously stored webhook with id %s", webhook_id)
+            new_webhook = await ctx.channel.create_webhook(name=webhook_name)
+            self.connection.update_webhook_id_of_record(channel_id=channel_id, webhook_id=new_webhook.id)
+            await ctx.send(f"""Updated webhook with name {webhook_name}.""")
+        else:
+            await ctx.send(f"""Text channel has not been setup with a webhook.
+                            Please run z!tc setup <name-of-webhook> to start using the z!tc bot.""")
+
+    @tc.command()
+    async def clear(self, ctx):
+        """
+        Clears the text channel of the webhook created by the twitcasting bot. Deletes any subscriptions added in this text channel.
+            Usage: z!tc clear
+        """
+        channel_id = f'{ctx.guild.id}:{ctx.channel.id}'
+        res = self.connection.select_records_by_id(channel_id=channel_id)
+        if res:
+            webhook_id = res[0]['webhook_id']
+            try:
+                self.connection.delete_all_records(channel_id=channel_id)
+                webhook = await self.bot.fetch_webhook(webhook_id)
+                await webhook.delete()
+                await ctx.send(f"""Cleared text channel of twitcasting subscriptions and webhook.""")
+            except NotFound:
+                self.logger.warning("Could not find and delete previously stored webhook with id %s", webhook_id)
+        else:
+            await ctx.send(f"""Text channel has not been setup with a webhook.
+                            Could not clear the text channel of any bot artifacts (if any).""")
+
+    @tc.command(hidden=True)
```

Comment:
should write a decorator for "admin" ~~whitelisted~~allowlisted commands and use this here, add a `TODO:` if you don't want to do it now